.teamcity/Gradle_Check/configurations/FunctionalTest.kt

```diff
@@ -29,7 +29,8 @@ class FunctionalTest(model: CIBuildModel, testCoverage: TestCoverage, subProject
                             + buildScanTags.map { buildScanTag(it) }
                             + buildScanValues.map { buildScanCustomValue(it.key, it.value) }
                     ).joinToString(separator = " "),
-            timeout = testCoverage.testType.timeout)
+            timeout = testCoverage.testType.timeout,
+            daemon = requiresDaemon(subProject))
```

Comment:
It seems to me that the decision whether or not run a functional test without the daemon belongs to the model (in this case I guess `FunctionalTestProject` would be the right place). WDYT?

Reply:
I think the correct model seems to be `GradleSubproject` so I put `requiresDaemon` there.

Reply:
Good call! I wonder if we should call the property `requiresDaemon`, though. How about `useDaemon`?